frontend/src/components/common/progress-details/ProgressDetails.tsx

```diff
@@ -161,11 +182,9 @@ const ProgressDetails = ({
           isButton
         />
       ) : (
-        <InfoBlock className={styles.notice}>
-          <Trans>
-            Delivery report has expired and is no longer available for download.
-          </Trans>
-        </InfoBlock>
+        <strong>
+          Delivery report has expired and is no longer available for download.\
```

Comment:
typo here

<img width="648" alt="image" src="https://user-images.githubusercontent.com/67887489/186153849-e54702a5-17c2-4f12-896f-1711a54fc062.png">

also is the grayed out text by design (I actually think no need, since it's no longer a button)